docs/source/documentation.rst

```diff
@@ -181,6 +113,19 @@ Below is a short tutorial describing how to use the AxonDeepSeg plugin for Napar
 
```

Comment:
I think because of the new section "Python Shell use", the sentence on line 110 makes a little less sense.

> Below is a short tutorial describing how to use the AxonDeepSeg plugin for Napari.

Maybe it could be good to directly link to the short tutorial, rather than saying "Below"?

```markdown
In the [Manual Correction](#manual-correction-of-segmentation-masks) section of the documentation, we provide a short tutorial describing how to use the AxonDeepSeg plugin for Napari.
```

Reply:
Change made - thanks!